src/cluster/src/start/k8.rs

```diff
@@ -1075,6 +1049,32 @@ impl ClusterInstaller {
         }
     }
 
+    /// Wait until the platform version of the cluster matches the chart version here
+    #[instrument(skip(self))]
+    async fn wait_for_fluvio_version(&self) -> Result<(), K8InstallError> {
```

Comment:
This should leverage `wait_for_sc_service` not creating another variant

Reply:
I was thinking about actually unifying all of the `wait_for_X` checks, but right now the project is not well fitted to do that. I am planning on revisiting this when I refactor, but for now I wanted to do the minimum to get it to work.

Reply:
I don't think this will work since it does't take account of ns